core/operation/src/main/java/uk/gov/gchq/gaffer/operation/OperationChain.java

```diff
@@ -66,7 +66,7 @@
 public class OperationChain<OUT> implements Output<OUT>,
         Operations<Operation> {
     private List<Operation> operations;
-    private Map<String, String> options;
+    private Map<String, String> options = new HashMap<>();
```

Comment:
This might change the string representation of JSON of operation chains. It might always display 
`"options":{}` which is verbose.

Reply:
See the discussion [in this comment](https://github.com/gchq/Gaffer/pull/3214#discussion_r1603314029), this is a good point though. It would likely be better to make further changes in the Proxy Store instead.

Reply:
Surely that would be better to have the options field displayed as you would hit the same issue where the key would be missing if trying to parse the JSON. It is arguably safer to prevent the null pointer from ever occurring than to expect every thing else to handle it.